.github/workflows/stable_docs.yml

```diff
@@ -0,0 +1,38 @@
+name: Update stable docs
+
+on:
+  push:
+    branches:
+      - 13.0.x
+    paths:
+      - documentation
+
+env:
+  target: git@github.com:infinispan/infinispan.github.io.git
+  target_branch: master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        cache: maven
+    - name: Build docs with Maven
+      run: mvn install -Pdistribution -pl documentation -am -s maven-settings.xml
+    - name: Copy docs to the site
+      run: |
+        git clone -b $target_branch $target
+        cp -r infinispan/documentation/target/generated/1*/html/* infinispan.github.io/docs/stable/
```

Comment:
It seems like the only difference between the two GH actions is the `infinispan.github.io` url and the commit message. We can just use a single action for both branches and just update the url depending on the branch.

https://stackoverflow.com/questions/58139406/only-run-job-on-specific-branch-with-github-actions